src/constants/android-settings.js

```diff
@@ -51,3 +51,57 @@ export const androidSettingsDefault = {
 }
 
 export const RESERVED_VALUES = 'reservedValues'
+export const SMS_TO_SEND = 'numberSmsToSend'
+export const SMS_CONFIRMATION = 'numberSmsConfirmation'
+
+export const generalSettings = {
```

Comment:
I think this variable could use a more descriptive name - it's not clear to me when seeing the name `generalSettings` that this contains the translated strings for the user-interface form elements.  Something like `generalSettingFormStrings` would be clearer, but there might even be a better option that I haven't thought of.